packages/extension/src/providers/ethereum/networks/unitzero.ts

```diff
@@ -0,0 +1,27 @@
+import icon from './icons/unitzero.webp';
+import wrapActivityHandler from '@/libs/activity-state/wrap-activity-handler';
+import { CoingeckoPlatform, NetworkNames } from '@enkryptcom/types';
+import { EtherscanActivity } from '../libs/activity-handlers';
+import { EvmNetwork, EvmNetworkOptions } from '../types/evm-network';
+
+
+const unitZeroOptions: EvmNetworkOptions = {
+  name: NetworkNames.UnitZero,
+  name_long: 'Unit Zero',
+  homePage: 'https://www.units.network/',
+  blockExplorerTX: 'https://explorer.unit0.dev/tx/[[txHash]]',
+  blockExplorerAddr: 'https://explorer.unit0.dev/address/[[address]]',
+  chainID: '0x15adb',
+  isTestNetwork: false,
+  currencyName: 'UNIT0',
+  currencyNameLong: 'Unit0',
+  node: 'https://rpc.unit0.dev/',
+  icon,
+  coingeckoID: 'units-network',
```

Comment:
Should this be unit0 instead? Based on the id on the coingecko page: https://www.coingecko.com/en/coins/unit0

Reply:
Fixed